react-app/src/app/screens/Game/components/Info/index.js

```diff
@@ -14,3 +15,16 @@ function InfoContainer({ history, status, onClick }) {
 }
 
 export default InfoContainer;
+
+InfoContainer.propTypes = {
+  history: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Try using:
```
PropTypes.arrayOf(PropTypes.shape({
   step: PropType...,
   move: PropType.....
}))
```

Reply:
This is because object is very generic. We want to know specifically which attributes does an object has. For this we use shape, it is not necessary to declare all. Just  the ones you are going to use here.

Reply:
Got it!